docs/4-user-guide/4-vscode.md

```diff
@@ -0,0 +1,26 @@
+# Getting started - VS Code
+
+Zarf uses an internally defined [schema](https://github.com/defenseunicorns/zarf/blob/master/zarf.schema.json) for its configuration files. This schema is used to validate the configuration files before they are used to build a Zarf package.
+
+## Adding Schema validation
```

Comment:
Schema should be lowercase or validation upper case (the below section uses `schema`)